src/main/kotlin/app/revanced/patches/idaustria/fingerprints/SpoofSignatureFingerprint.kt

```diff
@@ -0,0 +1,13 @@
+package app.revanced.patches.idaustria.fingerprints
+
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import org.jf.dexlib2.AccessFlags
+
+object SpoofSignatureFingerprint : MethodFingerprint(
+    "Ljava/lang/String",
```

Comment:
The full qualified name is not checked, only the first character. Same goes for any other descriptor.